### PR TITLE
qa: rotating node test on `v0.38.x`

### LINKED ISSUE #842: QA: rotating node test
Run the rotating node test as specified in `RELEASES.md` (using the infra existing in `qa-infra`), fix any found issue, compare with `v0.37.0` results. Finally populate the rotating node section of the `v0.38.x` report.

- [x] Set up the DO testbed
- [x] Run the test: loader set up using the saturation point found in the 200 node test
- [x] Obtain the relevant prometheus plots
  - Compare them to those obtained for `v0.37.0`
- [ ] Write corresponding section of the QA report
  - #883

DoD:

* No showstopper bugs found remain unresolved
* Rotating node section of the QA report is merged into `main` and `v0.38.x`
* No unacceptable performance regression detected (usually > 10% degradation w.r.t. `v0.37.0`

### PR DESCRIPTION
**Pull request by sergio-mena**
Closes #842

Section of the `v0.38.x` QA report on the rotating node.

[Rendered](https://github.com/cometbft/cometbft/blob/sergio/842-rotating/docs/qa/CometBFT-QA-38.md)

---

#### PR checklist

- [ ] Tests written/updated
- [ ] Changelog entry added in `.changelog` (we use [unclog](https://github.com/informalsystems/unclog) to manage our changelog)
- [ ] Updated relevant documentation (`docs/` or `spec/`) and code comments

### CODE REVIEW

File: docs/qa/CometBFT-QA-38.md [hunk 1]
```diff
@@ -238,7 +242,113 @@ commit versions used in the experiments.
...
+
+![rotating-all-latencies](img38/rotating/rotating_latencies.png)
+
+Which is similar to the baseline.
+
+![rotating-all-latencies](img37/200nodes_tm037/v037_rotating_latencies.png)
+
+The average increase of about 1 second with respect to the baseline is due to the higher
+transaction load produced (remember the baseline was affected by [\#9539], whereby most transactions
+produced were rejected by `CheckTx`)
```
Comment (lasarojc) [inline docs/qa/CometBFT-QA-38.md:269]:
```suggestion
produced were rejected by `CheckTx`).
```

File: scripts/qa/reporting/prometheus_plotter.py [hunk 2]
```diff
@@ -14,16 +15,21 @@
 
 from prometheus_pandas import query
 
-release = 'v0.37.x-alpha3'
+#release = 'v0.37.0-alpha.2'
```
Comment (lasarojc) [inline scripts/qa/reporting/prometheus_plotter.py:18]: wasn't it alpha.3?

File: docs/qa/CometBFT-QA-38.md [hunk 3]
```diff
@@ -238,7 +242,113 @@ commit versions used in the experiments.
...
+| 200-node | 2023-05-21 | v0.38.0-alpha.2 (1f524d12996204f8fd9d41aa5aca215f80f06f5e) | Pass   |
+
+
+## Rotating Node Testnet
+
+We use `c=1,r=400` as load, which can be considered a safe workload, as it was close to (but below)
+the saturation point in the 200 node testnet. This testnet has less nodes (10 validators and 25 full nodes).
+
+Unlike in the baseline tests, the version of CometBFT used for these tests is _not_ affected by [\#9539],
+which was fixed right after having run rotating testnet for `v0.37`.
```
Comment (lasarojc) [inline docs/qa/CometBFT-QA-38.md:254]: should stress here that for this test the baseline is alpha2 and not alpha3.

**Review by lasarojc — APPROVED** [review]

**Comment by lasarojc** [comment]
LGTM

**Review by hvanz — APPROVED** [review]
LGTM

**Pull request merged**